enforcer/src/main/resources/Dockerfile

```diff
@@ -40,7 +40,7 @@ ARG MOTD="\n\
 RUN \
     addgroup -S -g ${MG_USER_GROUP_ID} ${MG_USER_GROUP} \
     && adduser -S -u ${MG_USER_ID} -h ${MG_USER_HOME} -G ${MG_USER_GROUP} ${MG_USER} \
-    && mkdir ${MG_USER_HOME}/lib && mkdir -p ${MG_USER_HOME}/mg/security \
+    && mkdir ${MG_USER_HOME}/lib && mkdir -p ${MG_USER_HOME}/mg/security && mkdir -p ${MG_USER_HOME}/mg/droppings \
```

Comment:
I think we should add a folder to distribution (under enforcer directory and mount the path within docker-compose yaml) 
Let's get the opinion from others as well.

Reply:
Copied the droppings folder to the enforcer lib and mount the path within docker-compose.yaml